toolchain/check/context.h

```diff
@@ -735,10 +785,10 @@ class Context {
   // FinalizeImportRefBlock() will produce an inst block for them.
   llvm::SmallVector<SemIR::InstId> import_ref_ids_;
 
-  // Cache of allocated AnyBindName insts, keyed by the entity names they refer
-  // to. These are allocated while generating the pattern IR, but are emitted
-  // later as part of the pattern-match IR.
-  Map<SemIR::EntityNameId, SemIR::InstId> bind_name_cache_;
+  Map<SemIR::InstId, BindingPatternInfo> bind_name_map_;
+
+  // Stack of regions being being built as a result of BeginSubpattern calls.
```

Comment:
```suggestion
  // Stack of regions being built.
```
I think we expect this to be used in other ways too, right?